test/fixtures/output/kafka/mirror/truncate-and-restore-help.golden

```diff
@@ -0,0 +1,22 @@
+Truncate the local topic to the remote stopped mirror log end offsets and restore mirroring to the local topic to mirror from the remote topic.
+
+Usage:
+  confluent kafka mirror truncate-and-restore <local-topic-1> [local-topic-2] ... [local-topic-N] [flags]
+
+Examples:
+Truncate divergent records from mirror topics "my-topic-1" and "my-topic-2" and restore mirroring to the local topic to mirror from the remote topic:
```

Comment:
The example description here is different from the example description in the command_mirror_truncate_and_restore.go file, which will cause a pipeline test failure.

Reply:
Thanks for the comments @sgagniere :) .  Seems like the CI is failing again after the code change, anything I can do to fix it?

Reply:
From the semaphore build, it looks like we have to run `go mod tidy`?

```
ERRO Running error: context loading failed: no go files to analyze: running `go mod tidy` may solve the problem 
```

Reply:
To close the thread on this, the issue was because the incorrect tag was used for `kafkarest` sdk. Was fixed by @channingdong.